client/stylesheets/_toCat.scss

```diff
@@ -5167,7 +5173,7 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileUserFeedbackViewQuestion {
-    font-weight: 700;
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces